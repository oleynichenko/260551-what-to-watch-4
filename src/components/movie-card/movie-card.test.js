import MovieCard from "./movie-card.jsx";

const movie = {
  id: 1,
  title: `Теория большого взрыва: Откровение ринита`,
  image: `path`,
  preview: `path`,
};

it(`should render MovieCard with Big Bang Theory movie`, () => {
  const tree = renderer.create(
      <MovieCard
        movie={movie}
        onMovieCardMouseEnter={() => {}}
        onMovieCardMouseLeave={() => {}}
        onMovieTitleClick={() => {}}
        isVideoPlaying={false}
      />,
      {
        createNodeMock: () => {
          return {};
        }
      }
  ).toJSON();

  expect(tree).toMatchSnapshot();
});

