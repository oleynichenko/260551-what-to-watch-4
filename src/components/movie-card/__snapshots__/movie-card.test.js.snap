// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render MovieCard with Big Bang Theory movie 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <div
    className="small-movie-card__image"
  >
    <video
      className="player__video"
    />
  </div>
  <h3
    className="small-movie-card__title"
    onClick={[Function]}
  >
    <a
      className="small-movie-card__link"
      href="movie-page.html"
    >
      Теория большого взрыва: Откровение ринита
    </a>
  </h3>
</article>
`;
